service/notificationService.js

```
@@ -40,18 +40,19 @@ class NotificationService {
 	 */
 
 	formatNotificationMessage(monitor, status, platform, chatId) {
+
 		const messageText = this.stringService.getMonitorStatus(
-			monitor.name,
-			status,
-			monitor.url
+		  monitor.name,
+		  isUp,
```

ESLint

[error] 'isUp' is not defined.

(no-undef)